### PR TITLE
Command Line Arguments

### PR DESCRIPTION
**Pull request by vkbo**
Another small PR from my NiceToHave™ todo list is command line arguments. This is a feature of Fortran 2003.

The arguments are entirely optional, but this allows SixTrack to take the `fort.3` and `fort.2` files as arguments, meaning you no longer have to use those names. Of course, they both default to their current names.

I also added a few flags for echoing the version of the executable and just exit. These are:

```
$ ./sixtrack -v
SixTrack 5.2.4-59fb2f9

$ ./sixtrack -V
SixTrack
Version:  5.2.4
Released: 23.04.2019
Git Hash: 59fb2f99ba58d3f69a6856a1b75132ef63309c68

$ ./sixtrack -nv
50204
```

### CODE REVIEW

File: source/sixtrack.f90 [hunk 1]
```diff
@@ -663,7 +663,7 @@ subroutine daten
 
   case("RIPP") ! Power Supply Ripple Block
     write(lerr,"(a)") "INPUT> ERROR RIPP module has been removed and replaced by DYNK."
-    write(lerr,"(a)") "INPUT>       The script rippconvert.py in the pytools folder can be used to convert the fort.3 file."
+    write(lerr,"(a)") "INPUT>       The script rippconvert.py in the pytools folder can be used to convert the input file."
```
Comment (kyrsjo) [inline source/sixtrack.f90]: Is this message (pytools folder) still true?

Comment (vkbo) [reply source/sixtrack.f90]: No idea. It's your script isn't it?

Comment (kyrsjo) [reply source/sixtrack.f90]: I was just wondering, given the re-shuffling of all the files that have been going on.

```
15:28:26 kyrsjo@streifulv pytools $ ls
bdex_echo.py     fix_cdblocks.py  rippconverter.py  update_tests.py
compareSix2Madx  read_fortbin.py  SixRipken.py
15:28:26 kyrsjo@streifulv pytools $ pwd
/home/kyrsjo/code/SixTrack/source/pytools
```

Still there!

Comment (vkbo) [reply source/sixtrack.f90]: Good!

**Pull request merged**